react/AccordionItem/AccordionItem.less

```diff
@@ -0,0 +1,40 @@
+@import (reference) "~seek-style-guide/theme";
+
+.toggle {
+  position: absolute;
+  opacity: 0;
+}
+
+.toggleLink {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: @sk-link;
+  cursor: pointer;
+  min-height: (@grid-row-height * 4);
```

Comment:
Although the value is the same, please use `@row-height` as `@grid-row-height` is a legacy of the old grid system that we are hoping to retire at some stage.